.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: Create a bug report to help us improve the DepthAI
+title: "[BUG] {Title of the bug}"
+labels: bug
+assignees: ''
+
+---
+
+**Check if issue already exists**
+
+- Google it (e.g. *error xy* github luxonis depthai)
+- Check [troubleshooting](https://docs.luxonis.com/en/latest/pages/troubleshooting/) in documentation.
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Minimal Reproducible Example**
+Append the MRE to the bug report, [instructions here](https://docs.luxonis.com/en/latest/pages/support/#creating-minimal-reproducible-example)
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Pipeline Graph**
+
+Please also provide a screenshot of your pipeline using the [DepthAI Pipeline Graph](https://github.com/geaxgx/depthai_pipeline_graph).
+
+**Attach system log**
+ - Provide output of [log_system_information.py](https://github.com/luxonis/depthai/blob/main/log_system_information.py)
```

Comment:
Can you add the following additional things like 
Which OS, Which OS version and Which ROS version